package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -492,6 +493,11 @@ def __init__(self, filename, lammps_coordinate_convention="auto",
                              " is not a valid option. "
                              f"Please choose one of {option_string}")
 
+        if additional_columns:
+            self._additional_columns = additional_columns
+        else:
+            self._additional_columns = []
```

Comment:
What about using `additional_columns=[]` instead of `additional_columns=False` as default argument (an empty list evaluates to `False` anyways)?
```suggestion
        self._additional_columns = additional_columns
```

Reply:
I did it this way because of the "don't use empty lists as default keywords" debate, see [this](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments).

Reply:
Yes, but in this case it does not seem you are modifying the default list, aren't you?

However, if you want to be on the safe side, I think using `None` as default argument (as suggested by the article you linked) might be a better choice than `False`, so that future type hinting will become `Optional[list]` instead of `Union[bool, list]`.

Reply:
Yep you are right.

Ok, then I will do it like that, sounds good!